api/api-doc/collectd.json

```diff
@@ -67,7 +67,7 @@
                "parameters":[
                   {
                      "name":"pluginid",
-                     "description":"The plugin ID, describe the component the metric belongs to. Examples are cache, thrift, etc'. Regex are supported.The plugin ID, describe the component the metric belong to. Examples are: cache, thrift etc'. regex are supported",
+                     "description":"The plugin ID, describe the component the metric belongs to. Examples are cache, etc'. Regex are supported.The plugin ID, describe the component the metric belong to. Examples are: cache, etc'. regex are supported",
```

Comment:
This text is weird ("cache, etc." - what does it mean?) but because the previous version was weird too, I don't insist fixing it now.

Reply:
it is not only weird, but also redundant. this description actually repeats itself. so i

- replaced "thrift" with "alternator", which is another metric group, and 
- removed the duplicated description.